### PR TITLE
add multilabel text model wrapper

### PR DESCRIPTION
**Pull request by imatiach-msft**
Add wrapper for multilabel huggingface models in wrap_model method.

Given the scikit-learn docs here:

https://scikit-learn.org/stable/modules/multiclass.html?highlight=multilabel#multilabel-classification

The common wrapper will follow the format of the multioutput classifier:

https://scikit-learn.org/stable/modules/generated/sklearn.multioutput.MultiOutputClassifier.html#sklearn.multioutput.MultiOutputClassifier.predict

This format is a two-dimensional indicator array by num rows X num labels, where for index i = 1 if the corresponding label is output by the model:

https://scikit-learn.org/stable/modules/generated/sklearn.multioutput.MultiOutputClassifier.html#sklearn.multioutput.MultiOutputClassifier.predict

### CODE REVIEW

File: tests/common_text_utils.py [hunk 1]
```diff
@@ -30,6 +40,23 @@ def load_squad_dataset():
     return data
 
 
+def load_covid19_emergency_event_dataset():
+    dataset = datasets.load_dataset("joelito/covid19_emergency_event", split="train")
```
Comment (tongyu-microsoft) [inline tests/common_text_utils.py:44]: Just curious, is this a private dataset? Is there any privacy issue or any concern?

**Review by tongyu-microsoft — APPROVED** [review]

File: tests/common_text_utils.py (hunk 1 above)
Comment (imatiach-msft) [reply tests/common_text_utils.py:44]: no, there shouldn't be, it's a CC0 license dataset, which means it's in the public domain:
https://huggingface.co/datasets/joelito/covid19_emergency_event

see license info "cc0-1.0" on the dataset.

also on the github:
https://github.com/tommasoc80/COVID19_emergency_event/blob/main/LICENSE

"The Creative Commons CC0 Public Domain Dedication waives copyright interest in a work you've created and dedicates it to the world-wide public domain. Use CC0 to opt out of copyright entirely and ensure your work has the widest reach. As with the Unlicense and typical software licenses, CC0 disclaims warranties. CC0 is very similar to the Unlicense."
